server.js

```diff
@@ -40,14 +40,14 @@ app.get('/movies', function(req, res){
   res.json(movies);
 })
 
-//app.get('/', function(req, res, next) {   
-    //now you can call the get-driver, passing a callback function
-//    getMovies(function (err, moviesResult){ 
-       //you might want to do something is err is not null...      
-//       res.json(moviesResult);
+app.get('/', function(req, res, next) {   
+    // now you can call the get-driver, passing a callback function
+   getMovies(function (err, moviesResult){ 
+      //  you might want to do something is err is not null...      
+      res.json(moviesResult);
```

Comment:
what value is moviesResult taking?

Reply:
Please, can you explain me what means: // now you can call the get-driver, passing a callback function ?

Reply:
Sorry, unfortunately I'm not an expert on Js and his callback logic :(
The goal is that when we hit endpoint like / , call another function that takes care of the logic
IE: hitting /example, should call another function that do some logic and return what the example endpoint should return

Reply:
Please re-review my code. Several changes added!